tsdb/tsm1/encoding.go

```diff
@@ -25,6 +25,9 @@ const (
 	// BlockUnsigned designates a block encodes uint64 values.
 	BlockUnsigned = byte(4)
 
+	// blockUndefined represents an undefined block type value.
+	blockUndefined = BlockUnsigned + 1
```

Comment:
Can't think of a scenario but wondering if it would be safer to set `blockUndefined = byte(255)`, a value that won't be used for a block type in the future.

Reply:
I thought about that, but these constants currently must fit in 3 bits so it can be used for this fast look up:

https://github.com/influxdata/influxdb/pull/17563/files#diff-b44a6551f36995f10c237a575eec2bf7R737

It doesn't appear we're going to add more in this version of storage, so this constraint should hold true.